tools/dev/src/main/groovy/intellijRunConfig.groovy

```diff
@@ -92,7 +107,8 @@ containerNames.each{cn ->
                 env: encodeForXML(envMap),
                 sysProps : sysProps,
                 USER_HOME : '$USER_HOME$',
-                workingDir : getWorkDir(type)
+                workingDir : getWorkDir(type),
+                programParams: imageName.contains("controller") ? '0' : '--id  1'
```

Comment:
Any reason not to use `--id 0` for invoker? Running ansible OW locally, I run invoker0 only, and running invoker1 ends up with the multiple invoker situation where the down one doesn't go away etc. At least it is usable now (after the change to allow blackbox + managed invokers to be shared in small numbers), but for local simple runs, using a single invoker will always be simplest I think - so I would stick with `--id 0`.

Reply:
done